src/OpenSSL/SSL.py

```diff
@@ -699,8 +714,98 @@ def set_default_verify_paths(self):
 
         :return: None
         """
+        # This function will attempt to load certs from both a cafile and
+        # capath that are set at compile time. However, it will first check
+        # environment variables and, if present, load those paths instead
```

Comment:
Ah, ok, I see the ambiguity, "This function" refers to `SSL_CTX_set_default_verify_paths` but I thought it meant "the function we are in". Can you change the comment?